skyportal/models/comment.py

```diff
@@ -41,6 +41,8 @@ def backref_name(cls):
             return "comments"
         if cls.__name__ == 'CommentOnSpectrum':
             return 'comments_on_spectra'
+        if cls.__name__ == 'CommentOnGCN':
+            return 'comments_on_gcn'
```

Comment:
inconsistent naming here. On spectra its <plural>_on_<plural> whereas with GCN its <plural>_on_<singular>. You might change it to `comments_on_gcns`